src/migrations/00082-ensure-successful-releases-have-successful-images.sql

```diff
@@ -0,0 +1,21 @@
+DO $$
+BEGIN
+	-- Check that the DB is in a valid state and raise an exception if that's not the case
+  IF (
+    SELECT COUNT(*)
+    FROM "image" AS "image.0",
+      "release" AS "release.1",
+      "image-is part of-release" AS "image.0-is part of-release.1"
+    WHERE 'success' = "release.1"."status"
+    AND "release.1"."status" IS NOT NULL
+    AND "image.0-is part of-release.1"."image" = "image.0"."id"
+    AND "image.0-is part of-release.1"."is part of-release" = "release.1"."id"
+    AND NOT (
+      'success' = "image.0"."status"
+      AND "image.0"."status" IS NOT NULL
+    )
+  ) > 0
```

Comment:
Heads-up that the query part is using spaces while the rest of the file is using tabs.

Reply:
Nevermind I just did it and force pushed XD